project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.13
+sbt.version=1.4.9
```

Comment:
I think you can go as high as 1.5.1 and still have it build....

Reply:
only `1.4.9` is working